src/overlays/actors/ovl_En_Jsjutan/z_en_jsjutan.c

```diff
@@ -21,15 +27,455 @@ const ActorInit En_Jsjutan_InitVars = {
     (ActorFunc)EnJsjutan_Update,
     (ActorFunc)EnJsjutan_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Jsjutan/EnJsjutan_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Jsjutan/EnJsjutan_Destroy.s")
+// Shadow texture. 32x64 I8.
+u8 D_80A8E610[0x800];
+
+Vec3s D_80A8EE10[0x90];
+
+static s32 sUnused[2] = {
+    0,
+    0,
```

Comment:
```suggestion
    0
```
we probably don't need the formatter to do this